test/test_files/extension/extension.test

```diff
@@ -43,3 +43,15 @@ Binder exception: The extension /tmp/iceberg is neither an official extension, n
 ---- error
 Binder exception: Extension: fts is an official extension and has not been installed.
 You can install it by: install fts.
+
+-CASE ForceInstallExtension
+-STATEMENT INSTALL VECTOR;
+---- ok
+-STATEMENT INSTALL VECTOR;
+---- ok
+-STATEMENT FORCE INSTALL VECTOR;
+---- 1
+Extension: vector has been installed from repo: http://extension.kuzudb.com/.
+-STATEMENT UPDATE VECTOR;
+---- 1
+Extension: vector has been installed from repo: http://extension.kuzudb.com/.
```

Comment:
It'd be nicer if the message here says `Extension: vector **updated** from the ...`